packages/web/src.ts/geturl.ts

```diff
@@ -81,6 +89,7 @@ export async function getUrl(href: string, options?: Options): Promise<GetUrlRes
 
         method: (options.method || "GET"),
         headers: shallowCopy(options.headers || { }),
+        agent: (agent || false),
     };
```

Comment:
I remember that `@types/node` checks that this should only be one of the `Agent` supported by `http` or otherwise it is a boolean.